libraries/GCS_MAVLink/GCS.h

```diff
@@ -79,6 +79,7 @@ enum ap_message {
     MSG_BATTERY_STATUS,
     MSG_AOA_SSA,
     MSG_LANDING,
+    MSG_SUB_INFO,
```

Comment:
Why not MSG_NAMED_FLOAT instead? It would allow other vehicles to use it if they wanted.